apps/meteor/client/views/admin/subscription/SubscriptionPage.tsx

```diff
@@ -130,16 +130,17 @@ const SubscriptionPage = () => {
 						)}
 						<Box marginBlock='none' marginInline='auto' width='full' color='default'>
 							<Grid m={0}>
-								<Grid.Item lg={4} xs={4} p={8}>
+								<Grid.Item lg={4} xs={4} p={8} minHeight={244}>
```

Comment:
Since the grid has a padding of 8px, you might want to consider increasing the minHeight to 260px (244 + 16). Adding the minHeight 244px directly to the `FeatureUsageCard` would also work (and wouldn't break if the padding changes in the future).

Reply:
Adding the minHeight to FeatureUsageCard isn't really an option for now as it would require changing fuselage since the Card component doesn't have box style props. I'll change the minHeight of the grid item to 260